.gitignore

```diff
@@ -3,7 +3,6 @@ target/
 [._]*.sw[a-p]
 **/*.rs.bk
 Cargo.lock
-Cargo.toml
```

Comment:
What's the reason for this change? The root `Cargo.toml` should never be added to git.